registrations/tasks.py

```diff
@@ -265,6 +265,34 @@ def validate(self, registration):
             self.l.info("Registration object updated.")
             return False
 
+    def create_popi_subscriptionrequest(self, registration):
+        """
+        Creates a new subscription request for the POPI message set. This
+        message set tells the user how to access the POPI required services.
+        This sould only be sent for Clinic or CHW registrations.
+        """
+        if ('prebirth' not in registration.reg_type or
+           registration.source.authority not in ['hw_partial', 'hw_full']):
+            return "POPI Subscription request not created"
+
+        self.l.info("Fetching messageset")
+        msgset_short_name = utils.get_messageset_short_name(
+            'popi', registration.source.authority, None)
```

Comment:
Yes, that was the plan.

I thought of modifying the short name function, but the convention for naming the message sets is `<name>.<authority>.<number>`, and I'm not sure what to put in the `<authority>` bit if we make it the same message set, and I'd like to stick to the naming convention.